install-sphinx-cron.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+(crontab -l ; echo "* * * * * /usr/local/bin/docker exec -t sphinx bash /root/index.delta.sh") | crontab -
+
+(crontab -l ; echo "*/5 * * * * /usr/local/bin/docker exec -t sphinx bash /root/index.all.sh") | crontab -
```

Comment:
Does this have to run on the user's localhost? I think it would make a lot more sense to run in the container, but I know you mentioned issues getting that up and running. What issues were they, specifically? I did some *very* brief testing and this seemed to at least get cron installed in the container.

```
yum install cronie
crontab -e
crontab -l
```

Reply:
I've spend few days trying to get `yum install -y cronie` to our sphinx image. But failed.
Error messages like:
`Rpmdb checksum is invalid: dCDPT(pkg checksums):`
So I dropped that, probably will switch to another base sphinx container not from
`FROM leodido/sphinxsearch:2.2.10`